tests/migration/migration.go

```diff
@@ -3025,11 +3025,13 @@ var _ = SIGMigrationDescribe("VM Live Migration", func() {
 			By("reserving the cores used by the VMI on the second node with a paused pod")
 			var pods []*k8sv1.Pod
 			var pausedPod *k8sv1.Pod
+			ns := testsuite.GetTestNamespace(pausePod)
 			libnode.AddLabelToNode(nodes[1].Name, testLabel2, "true")
-			for pausedPod = libpod.RunPod(pausePod); !hasCommonCores(vmi, pausedPod); pausedPod = libpod.RunPod(pausePod) {
+			for pausedPod, err = libpod.Run(pausePod, ns); !hasCommonCores(vmi, pausedPod) && err == nil; pausedPod, err = libpod.Run(pausePod, ns) {
```

Comment:
Ill just say that the following 10 lines are pretty bad and unclear, why someone creates a list of pods just to delete them? why not instead of append do the delete? I wont ask you to change that, only if you really want lol